bat/tests/customize-os-release/run.bats

```diff
@@ -21,11 +21,13 @@ setup() {
   #Compare the diff between customized os-release file and output of mixer build
   #bundle. The only difference should be VERSION_ID 
   run bash -c "diff files/my-os-release update/image/10/full/usr/lib/os-release | wc -l "
-  [ "$output" -eq  4 ]
+  [ "$output" -eq  6 ]
 
   run bash -c "diff files/my-os-release update/image/10/full/usr/lib/os-release"
   [[ ${lines[1]} =~ "VERSION_ID" ]]
-  [[ ${lines[3]} =~ "VERSION_ID" ]]
+  [[ ${lines[4]} =~ "VERSION_ID" ]]
+  [[ ${lines[2]} =~ "BUILD_ID" ]]
+  [[ ${lines[5]} =~ "BUILD_ID" ]]
```

Comment:
These tests also fail for me. With the following changes, they pass:

```
  [[ ${lines[1]} =~ "VERSION_ID" ]]
  [[ ${lines[3]} =~ "VERSION_ID" ]]
  [[ ${lines[5]} =~ "BUILD_ID" ]]
  [[ ${lines[7]} =~ "BUILD_ID" ]]
```

Reply:
@jwakre updated.